### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by alia-peterson** [alia-peterson/whats-cooking#43]
**Description of what needs refactoring**
Should be able to tab through entire website and access every item
Should be able to use website with a screen reader
Website should still be usable with a colorblindness extension 

**Additional Context** 
Need to check that a screen reader will convey that the pantry is a hidden object that can be interacted with
